client/src/components/profile-page/EditContent.js

```diff
@@ -214,14 +225,14 @@ export default function EditContent() {
           </div>
           {isClicked ? (
             <div className="edit-page-buttons-wrapper">
-              <button className="save-button" disabled>
+              <button className="generic-button" disabled>
```

Comment:
This logic connected to `isClicked` should only determine the component's attributes, not these two long segments of JSX.
Just because that ternary is really difficult to read like it is right now